examples/null.md

```diff
@@ -0,0 +1,141 @@
+# Null primitive type
+
+The `nullable` type attribute can be used to outline that `null` is an accepted value.
+
+## Using `null` with `string` and `array` types
+
+### MSON
+
+```mson
+- keyA (string, nullable, required) - Accepts `null` or `string`
+- keyB (array, nullable, required) - Accepts `null` or an `array`
+```
+
+### JSON
+
+```json
+{
+   "keyA": null,
+   "keyB": null
+}
+```
+
+```json
+{
+   "keyA": "bob",
+   "keyB": [1, 2, 3]
+}
+```
+
+## Using `null` as a possible value for an array
+
+### MSON
+
+```mson
+- keyA (array[enum], required) Accepts an array of `null` or `string`
+    - (string, required)
+```
```

Comment:
@pksunkara Would you like me to re-write this or simply remove it? Not sure exactly how valuable it is. The idea was to outline how to have a required array that may accept null types however I see the issues with it now.